libs/lib-linker/src/lib/consolidation-result/consolidation-result.component.html

```diff
@@ -12,9 +12,16 @@
       <h1>
         {{('SHARED_LIB.CONSOLIDATOR.' + result + '_TITLE') | translate}}
       </h1>
-      <h5>
+      <h5 *ngIf="!(userMail && result === 'OK')">
         {{('SHARED_LIB.CONSOLIDATOR.' + result + '_SUBTITLE') | translate}}
-        <span *ngIf="result === 'UNKNOWN_ERROR'"> {{supportMail}} </span>
+        <span *ngIf="result === 'UNKNOWN_ERROR' || result === 'MESSAGE_SENT_TO_SUPPORT'">
+          {{supportMail}}
+        </span>
```

Comment:
oh sorry, you misunderstood me
I wanted to show the user to what HIS email will support contact him - so not the perun@cesnet.cz but -> eg. luckureckova@gmail.com

Reply:
Oh my bad, I changed the email and text for this use case (MESSAGE_SENT_TO_SUPPORT).